drivers/spi/spi_rpi_pico_pio.pio

```diff
@@ -0,0 +1,78 @@
+; Copyright (c) 2023 Stephen Boylan <stephen.boylan@beechwoods.com>
```

Comment:
Is this file needed?

Reply:
I think these codes are unnecessary as it is already embedded in the C source as a comment.

Reply:
@ThreeEights 
Please remove this file.

Reply:
Removed.